src/ClientResumeStatusCallback.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#pragma once
+
+#include <folly/ExceptionWrapper.h>
+
+namespace reactivesocket {
+
+class ClientResumeStatusCallback {
```

Comment:
class with virtual methods should have a virtual destructor. That is a lint rule in FB...